timemachine/fe/free_energy.py

```diff
@@ -37,6 +38,16 @@ class MDParams:
     n_frames: int
     n_eq_steps: int
     steps_per_frame: int
+    local_steps: int = 0
+    k: float = 10_000.0
+    min_radius: float = 1.0
+    max_radius: float = 3.0
+    freeze_reference: bool = True
+
+    def __post_init__(self):
+        assert self.min_radius <= self.max_radius
+        assert self.local_steps <= self.steps_per_frame
+        assert self.k > 0.0
```

Comment:
nit: strengthen to match assertions in local_md_utils.cu

(1.0 < self.k < 1e6, self.min_radius > 0.1)

Reply:
Strengthened assertions to match and expanded tests in https://github.com/proteneer/timemachine/pull/1026/commits/29680b9a2d6ff6c72d388b85c38a5da892b69161